src/Altinn.Platform/Altinn.Platform.Receipt/Receipt/Configuration/PlatformSettings.cs

```diff
@@ -1,19 +1,58 @@
+using System;
+
 namespace Altinn.Platform.Receipt.Configuration
 {
     /// <summary>
     /// Configuration for platform settings
     /// </summary>
     public class PlatformSettings
     {
-        /// <summary>
-        /// Gets or sets the url for the API Profile endpoint
-        /// The config value was needed for test purposes
-        /// </summary>
-        public string ApiProfileEndpoint { get; set; }
+        private string _subscriptionKey;
+        private string _apiStorageEndpoint;
+        private string _apiRegisterEndpoint;
+        private string _apiProfileEndpoint;
 
         /// <summary>
         /// Gets or sets the url for the API Authentication endpoint
         /// </summary>
         public string ApiAuthenticationEndpoint { get; set; }
+
+        /// <summary>
+        /// Gets or sets the url for the API profile endpoint
+        /// </summary>
+        public string ApiProfileEndpoint
+        {
+            get => Environment.GetEnvironmentVariable("PlatformSettings__ApiProfileEndpoint") ?? _apiProfileEndpoint;
```

Comment:
When we have `config.AddEnvironmentVariables();` in the Program.cs file the values from Environment Variables are added to config for us. Making this type of logic unnecessary. Just verify that the name of the Environment Variable is correct. Section name must match and the setting name must match, separated with two _ characters.

Reply:
done